onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,113 @@
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+def fetch_user_formbuilder_details(self, user):
+    with connection.cursor() as cursor:
+        cursor.execute(
+            "select * from hub_extrauserdetail where user_id={}".format(
+                user.pk))
+        cursor.execute(
+            "select * from hub_perusersetting where name={}".format(
```

Comment:
@WinnyTroy why do we have to do raw SQL here?

Reply:
@moshthepitt  this was intentional in an attempt to query kpi's table for records as Onadata is unaware of the model instances of that table.
An alternative could be to hit the assets endpoint endpoints to return that data. Unfortunately the endpoint does not allow `DELETE` as a valid request method. So raw sql queries seemed the cleanest altternative